lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -92,6 +92,14 @@ def self.instance_name
       "github"
     end
 
+    # @!group PR Review
+    # Current PR Review object.
+    # @return [GitHubReview]
+    #
```

Comment:
Sure thing, something like this

```
# @!group PR Review
#
# In Beta. Provides access to creating a GitHub Review instead of a typical GitHub comment.
#
# To use you announce the start of your review, and the end via the `start` and `submit` functions,
# for example:
#
# ```
#   github.review.start
#   github.review.fail("Please add a CHANGELOG entry") if has_no_changelog
#   github.review.warn("Highway to the Danger Zone") if pr_includes_word_danger
#   github.review.message("You might want to read #{url}") if may_require_docs
#   github.review.submit
# ```
#
# @return [GitHubReview]
```